src/views/BreadView/BreadTable.js

```diff
@@ -0,0 +1,178 @@
+        );
+    })
+  }
+
+  handleAddToOrder = value => {
+    this.setState(
+        {dialogOpen: false, submitting: true},
+        () => this.doAddItem(value)
+    );
+  }
+
+  doAddItem = value => {
+    api.patch({
+        path: '/bread/' + this.state.order.id,
+        data: { items: [value] },
+    }).then(data => {
+        this.setState(
+          {submitting: false, order: {}},
+          () => this.props.showMessage('Brood toegevoegd', this.props.refresh)
+        );
+    }).catch(error => {
+        this.setState(
+          {submitting: false, order: {}},
+          () => this.props.showMessage(error.message)
+        );
+    })
+  }
+
+  handleDialogChange = (dialogOpen) => () => {
+    this.setState({dialogOpen: dialogOpen});
+  }
+
+  render () {
+    const { classes, orders, loading, items } = this.props;
+
+    return (
+        <React.Fragment>
+          <Table>
+            <TableHead>
+              <TableRow>
+                <TableCell className={classes.dateCell}>Week</TableCell>
+                <TableCell className={classes.ordersCell}>Bestelling</TableCell>
+                <TableCell className={classes.buttonCell}></TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {loading ? emptyRow(classes) : null}
+              {!loading && orders.length === 0 ? emptyRow(classes, 'Geen bestellingen mogelijk') : null}
+              {orders.map(row => {
+                 return (
+                   <TableRow key={row.id}>
+                     <TableCell className={classes.dateCell}>
+                       {utils.formatDate(row.date)}
+                     </TableCell>
+                     <TableCell className={classes.ordersCell}>
+                         {row.items.map(item => item.name).join(", ")}
+                         {/*  {row.items.map(item => item.price/100).reduce((a,b) => a + b, 0)} */}
+                     </TableCell>
+                     <TableCell className={classes.buttonCell}>
+                        <IconButton
+                           title="Voeg brood toe"
+                           disabled={this.state.submitting}
+                           onClick={this.handleAddClick(row)}
+                        >
+                           { this.state.submitting && row.id === this.state.order.id
+                               ? <CircularProgress size={20} />
+                               : <AddIcon fontSize="small" />
+                           }
+                        </IconButton>
+                        <IconButton
+                           title="Verwijder bestelling"
+                           disabled={this.state.submitting}
+                           onClick={this.handleClearOrder(row)}
+                        >
+                           { this.state.submitting && row.id === this.state.order.id
+                               ? <CircularProgress size={20} />
+                               : <ClearIcon fontSize="small" />
```

Comment:
ClearIcon -> DeleteIcon?

Reply:
ik zou toch graag de notie van "bestelling leegmaken" in plaats van "bestelling onherroepelijk verwijderen" houden

Reply:
oke, delete moet dan niet, maar het is zeker nooit 'onherroepelijk verwijderen he'? je kan altijd opnieuw de bestelling doen he?

Reply:
Ja, klopt. Zolang alle weken bestaan kun je altijd nieuwe bestellingen plaatsen.